static/app/utils/replays/replayDataUtils.tsx

```diff
@@ -270,14 +283,27 @@ export function replayTimestamps(
   const spanStartTimestamps = rawSpanDataFiltered.map(span => span.startTimestamp * 1000);
   const spanEndTimestamps = rawSpanDataFiltered.map(span => span.endTimestamp * 1000);
 
+  // Calculate min/max of each array individually. This prevents two things:
+  // - Avoid extra array allocations
+  // - Avoid `Maximum call stack size exceeded` when the array is too large for Math.min\Math.max
```

Comment:
Oh I see. Maybe we can explain with a docblock comment on the `getMinMax` function why we need it.